src/api/OrderAPI.ts

```diff
@@ -158,6 +163,7 @@ export class OrderAPI {
    * @param  makerRelayerFee             Amount of relayer token paid to relayer by maker
    * @param  takerRelayerFee             Amount of relayer token paid to relayer by taker
    * @param  salt                        Optional salt to include in signing
+   * @param  isMetamaskSigner            Optional parameter if the signature is generated by metamask
```

Comment:
maybe name it `addSignaturePrefix` since we don't know if any other client might have the same issue.

Reply:
+1